LinqToRest.Test/QueryProviderTest.cs

```diff
@@ -13,66 +14,209 @@ public class QueryProviderTest
         [Fact]
         public void ReturnsRestQuery()
         {
-            var query = CreateQuery<EntityWithQueryableMember>();
-            var restQuery = query.ToString();
-            var expectedRestQuery = $"{MockServiceUri().AbsoluteUri}entitywithqueryablemembers";
+            var actual = CreateQuery<EntityWithQueryableMember>()
+                .ToString();
 
-            Assert.Equal(expectedRestQuery, restQuery);
+            var expected = EntityWithQueryableMemberResult;
+
+            Assert.Equal(expected.Query, actual);
         }
 
-        private static Query<T> CreateQuery<T>()
+        [Fact]
+        public void ReturnsRestQueryWithSelect()
         {
-            var serviceUri = MockServiceUri();
-            var resourceRetriever = MockResourceRetriever();
-            var queryBinderFactory = MockQueryBinderFactory();
-            var queryProvider = new QueryProvider(resourceRetriever, queryBinderFactory, serviceUri);
+            var actual = CreateQuery<EntityWithQueryableMember>()
+                .Select(entity => new { entity.Name })
+                .ToString();
 
-            return new Query<T>(queryProvider);
+            var expected = UniqueIdentifierNameResult;

```

Comment:
Move `.Query ` to `var expected` evrywere in file

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/3cfbe26f2b65048b507dff21c7b572f2d5a5ae1c